go-controller/pkg/ovn/controller/services/utils.go

```diff
@@ -188,7 +217,12 @@ func createPerNodePhysicalVIPs(isIPv6 bool, protocol v1.Protocol, sourcePort int
 			return err
 		}
 
-		// If self ip is in target list, we need to use special IP to allow hairpin back to host
+		newEps := eps
```

Comment:
I know I am reviewing myself :-), but this can go in the if now. We only care about overriding targetIPs.
I'd also drop a log there

Reply:
ACK